SingularityExecutor/src/main/resources/deploy.env.hbs

```diff
@@ -1,7 +1,13 @@
 # mesos task env
+{{#if docker}}
+{{#each env}}
+{{{name}}}={{{bashEscaped value}}}
```

Comment:
it can't read it that way. The docker.env file can only be read as KEY=VAL, although within the value it will act like bash (as far as i know)